Makefile.am

```diff
@@ -1496,7 +1496,6 @@ sssd_secrets_SOURCES = \
 sssd_secrets_LDADD = \
     $(HTTP_PARSER_LIBS) \
     $(JANSSON_LIBS) \
-    $(TDB_LIBS) \
     $(SSSD_LIBS) \
```

Comment:
NACK

```
sh$ objdump -T .libs/sssd_secrets | grep TDB
0000000000000000      DF *UND*  0000000000000000  TDB_1.2.1   tdb_open
0000000000000000      DF *UND*  0000000000000000  TDB_1.2.1   tdb_delete
0000000000000000      DF *UND*  0000000000000000  TDB_1.2.1   tdb_traverse
0000000000000000      DF *UND*  0000000000000000  TDB_1.2.1   tdb_store
0000000000000000      DF *UND*  0000000000000000  TDB_1.2.1   tdb_errorstr
0000000000000000      DF *UND*  0000000000000000  TDB_1.2.1   tdb_fetch
```

We should do opposite. Remove `SSSD_LIBS` and replace with required libraries
```
sh$ objdump -p .libs/sssd_secrets | grep NEEDED
  NEEDED               libhttp_parser_strict.so.2
  NEEDED               libjansson.so.4
  NEEDED               libtevent.so.0
  NEEDED               libtalloc.so.2
  NEEDED               libpopt.so.0
  NEEDED               libldb.so.1
  NEEDED               libdbus-1.so.3
  NEEDED               libdhash.so.1
  NEEDED               libselinux.so.1
  NEEDED               libtdb.so.1
  NEEDED               libsystemd.so.0
  NEEDED               libsss_util.so
  NEEDED               libsss_crypt.so
  NEEDED               libsss_debug.so
  NEEDED               libcurl.so.4
  NEEDED               libc.so.6
```

Because there is no reason why `sssd_secrets` shoudl be linked with `PCRE` or `openldap` libs which are part of `SSSD_LIBS`